src/components/loading-indicator/LoadingIndicator.scss

```diff
@@ -0,0 +1,88 @@
+@import '../../styles/variables';
+
+.hui-LoadingIndicator {
+    position: relative;
+}
+
+.hui-LoadingIndicator-overlay {
+    display: flex;
+    justify-content: center;
+
+    .hui-LoadingIndicator--bottom &,
+    .hui-LoadingIndicator--middle &,
+    .hui-LoadingIndicator--top & {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+    }
+
+    .hui-LoadingIndicator--bottom & {
+        align-items: flex-end;
+    }
+
+    .hui-LoadingIndicator--middle & {
+        align-items: center;
+    }
+
+    .hui-LoadingIndicator--top & {
+        align-items: flex-start;
+    }
+
+    .hui-LoadingIndicator.hui-is-blurred & {
+        background-color: rgba($hui-gray-02, 0.75);
+    }
+
+    .hui-LoadingIndicator.hui-is-hidden & {
+        background-color: $hui-white;
+    }
+}
+
+// TODO: Consider an alternative to `transform` for resizing the animation
```

Comment:
The new height of `hui-LoadingIndicator-pulse` is not respected by the parent container when using `transform`

Reply:
The outer div (`hui-LoadingIndicator-overlay`) can be removed if the `width` and `height` are fixed values